src/VisualStudio/CSharp/Impl/CodeModel/CSharpCodeModelService.cs

```diff
@@ -1967,6 +1968,34 @@ public override SyntaxNode SetParameterKind(SyntaxNode node, EnvDTE80.vsCMParame
             return parameter.WithModifiers(newModifiers);
         }
 
+        public override EnvDTE80.vsCMParameterKind UpdateParameterKind(EnvDTE80.vsCMParameterKind parameterKind, PARAMETER_PASSING_MODE passingMode)
+        {
+            var updatedParameterKind = parameterKind;
+
+            switch (passingMode)
+            {
+                case PARAMETER_PASSING_MODE.cmParameterTypeIn:
+                    updatedParameterKind |= EnvDTE80.vsCMParameterKind.vsCMParameterKindNone;
```

Comment:
why the ors and ands.  Why not just say if it's cmParameterTypeIn then parameter kind is vsCMParameterKindNone?